source/extensions/filters/http/jwt_authn/jwks_cache.cc

```diff
@@ -36,10 +36,9 @@ class JwksDataImpl : public JwksCache::JwksData, public Logger::Loggable<Logger:
     }
     audiences_ = std::make_unique<::google::jwt_verify::CheckAudience>(audiences);
     bool enable_jwt_cache = jwt_provider_.enable_jwt_cache();
-    int jwt_cache_size = jwt_provider_.jwt_cache_size();
-    tls_.set([enable_jwt_cache, jwt_cache_size](Envoy::Event::Dispatcher& dispatcher) {
-      return std::make_shared<ThreadLocalCache>(enable_jwt_cache, jwt_cache_size,
-                                                dispatcher.timeSource());
+    const auto& config = jwt_provider_.jwt_cache_config();
+    tls_.set([enable_jwt_cache, config](Envoy::Event::Dispatcher& dispatcher) {
+      return std::make_shared<ThreadLocalCache>(enable_jwt_cache, config, dispatcher.timeSource());
```

Comment:
bool enable_jwt_cache = jwt_provider_.has_jwt_cache_config();

Reply:
Do you want to remove `enable_jwt_cache` from `JwtProvider`?

Reply:
Yes,   by creating jwt_cache_config proto message implicitly enables jwt_cache.

Reply:
Removed `enable_jwt_cache`